config/RConfigure.in

```diff
@@ -36,6 +36,7 @@
 #@hasstdinvoke@ R__HAS_STD_INVOKE /**/
 #@hasstdindexsequence@ R__HAS_STD_INDEX_SEQUENCE /**/
 #@has_found_attribute_always_inline@ R__HAS_ATTRIBUTE_ALWAYS_INLINE /**/
+#@has_found_attribute_no_inline@ R__HAS_ATTRIBUTE_NO_INLINE /**/
```

Comment:
`R__HAS_ATTRIBUTE_NOINLINE`?

Reply:
the underscore was following the pattern of  always_inline ...